customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDto.java

```diff
@@ -231,6 +233,14 @@ public void setSector(Sector sector) {
         this.sector = sector;
     }
 
+    public List<PublicationTypes> getPublicationTypes() {
+        return publicationTypes;
+    }
+
+    public void setPublicationTypes(List<PublicationTypes> publicationTypes) {
```

Comment:
For user-friendliness, make the argument a `Collection<PublicationTypes>`, assigning via `new HashSet(publicationTypes)`